Clarithromycin/Clarithromycin_evaluation_report.md

```diff
@@ -1,12 +1,16 @@
+
+
+
+
 # Building and evaluation of a PBPK model for clarithromycin in healthy adults
 
 
 
 
 
-| Version                                         | 1.2-OSP12.0                                                   |
+| Version                                         | 1.0-OSP12.0                                                   |
 | ----------------------------------------------- | ------------------------------------------------------------ |
-| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Clarithromycin-Model/releases/tag/v1.2 |
+| based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Clarithromycin-Model/releases/tag/v1.0 |

```

Comment:
Link to the model tag will be correct in the final report; here it is 1.0 just for testing